### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### The following changes have been made to this PR:

- Created a database named vet_clinic.
- Created a table named animals
- Inserted the provided data
- Write the queries for the provided quests
- Screenshots Attached
image
image
image
image
image
image
image
image

Thank you :)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+
+SELECT name
+FROM animals
+WHERE
+    date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
```
Comment (BenMukebo) [inline queries.sql]: - [x] It must list the name of all animals born between 2016 and 2019. you should consider all the days of 2019 and not only the first day of January 2019. Your code might looks like this:

```SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';```
or
```SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Metaverse-Nabeel,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- GitHub flow was correctly used.  ✔️   
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Metaverse-Nabeel) [reply queries.sql]: Hi @BenMukebo 

Thank you for your valuable feedback. I shall make the required changes.

Regards

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Metaverse-Nabeel 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Animals table is created perfect!
-  ✅ Great work using the right Gitflow 
-  ✅ Good job with queries

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**